Misc/NEWS.d/next/Library/2017-11-07-19-12-25.bpo-31955.1DWu-S.rst

```diff
@@ -0,0 +1,5 @@
+Fixed incorrect discrimination between string and list types in
+distutils.CCompiler.set_executables() method. Previously if unicode string
+was passed as executable value it was considered not as string, but as list.
+It caused TypeError: "coercing to Unicode: need string or buffer, list
+found" at unixccompiler.py, line 122, in _compile().
```

Comment:
NEWS entries are read by end users who don't care much to details. I suggest to be much more concise. For example:

Fix CCompiler.set_executable() of distutils to handle properly Unicode strings.